samples/ehr-service/lambda/callback/index.js

```diff
@@ -0,0 +1,87 @@
+import AWS from 'aws-sdk';
+import fetch from 'node-fetch';
+
+const apiEndpoint = process.env.API_ENDPOINT
+const clientKey = process.env.CLIENT_KEY
+const clientSecret = process.env.CLIENT_SECRET
+
+AWS.config.update({ region: process.env.AWS_REGION || 'us-west-1' });
+
+export async function handler(event, context) {
+  console.log("Received event:", JSON.stringify(event, null, 2));
+  const queryStringParameters = event.queryStringParameters;
+  let body = "Invalid request";
+  let statusCode = "403";
+  let headers = {
```

Comment:
Please remove this. We don't need CORS in this instance as console.katanemo.com does a HTTP 302 redirect not via XHTMLHttpRequest